ms/MSOper/MSMetaData.h

```diff
@@ -564,7 +564,23 @@ class MSMetaData {
 
     const MeasurementSet* getMS() const { return _ms; }
 
-protected:
+    // If set to true, enable parallel computations where possible,
+    // else, always do serial computations.
+    void setParallel(Bool b) {
```

Comment:
this needs to be implemented in the .cc file, when its in the header the user of the header has to enable openmp regardless on how casacore itself is built.